pep-0001.txt

```diff
@@ -379,7 +379,7 @@ optional and are described below.  All other headers are required. ::
   * Requires: <pep numbers>
     Created: <date created on, in dd-mmm-yyyy format>
   * Python-Version: <version number>
-    Post-History: <dates of postings to python-list and python-dev>
+    Post-History: <dates of postings to python-dev and/or python-ideas>
```

Comment:
Actually, this needs to be `python-ideas and/or python-dev`.  The reasoning is that python-dev must always be included but python-ideas is optional.

Reply:
Same comment below.